File: A1/CSHARP/GeradordeMatrizes/MatrizF.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography;
using System.Text;
using System.Threading.Tasks;

namespace CSHARP.GeradordeMatrizes
{
    internal class MatrizF
    {
        public float[,] matrizF = new float[3, 3];
        public float[,] getMatrizF()
        {
            return matrizF;
        }

        public void setMatrizF()
        {
            this.matrizF = new float[3, 3] {
                { GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita() },
                { GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita() },
                { GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita() }
            };
        }

        static float GeradordeImprobabilidadeInfinita()
        {
            Random aleatorio = new();
            float valor = float.Parse((aleatorio.NextDouble() * 100).ToString());
            return valor;
        }

        public static void printaMatriz(float[,] matriz)
        {
            for (int x = 0; x < 3; x++)
            {
                for (int y = 0; y < 3; y++)
                {
                    if (y != 2)
                    {
                        Console.Write(matriz[x, y]);
                        Console.Write("  ");
                    }
                    else
                    {
                        Console.Write(matriz[x, y]);
                        Console.Write("\n");
                    }
                }
            }
        }

        public static void multiplicaMatrizFLOAT(float[,] matriz1, float[,] matriz2)
        {
            float[,] matrizRes = new float[3, 3];

            for (int x = 0; x < 3; x++)
            {
                for (int y = 0; y < 3; y++)
                {
                    matrizRes[x, y] = 0;
                    for (int z = 0; z < 3; z++)
                    {
                        matrizRes[x, y] += matriz1[x, z] * matriz2[z, y];
                    }
                }
            }
            printaMatriz(matrizRes);
        }
    }
}

File: A1/Java/Matrizes/src/GeradoresdeMatrizes/MatrizF.java
package GeradoresdeMatrizes;

import java.util.Random;

public class MatrizF {
	
	
	public float[][] matrizF = { 
			{ GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita() },
			{ GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita() },
			{ GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita(), GeradordeImprobabilidadeInfinita() } 
	};
	
	public float[][] getMatrizF() {
		return matrizF;
	}

	public void setMatrizF(float[][] matrizF) {
		this.matrizF = matrizF;
	}
	
	
	
	public static void printaMatriz(float[][] matriz) {
		for (int x = 0; x < 3; x++) {
			for (int y = 0; y < 3; y++) {
				if (y != 2) {
					System.out.printf(" %2f ", matriz[x][y]);
				} else {
					System.out.printf(" %2f \n", matriz[x][y]);
				}
			}
		}
	}
	
	
	
	static float GeradordeImprobabilidadeInfinita() {
		Random aleatorio = new Random();
		float valor = aleatorio.nextFloat(100);
		return valor;
	}

	
	
	public static void multiplicaMatrizFLOAT(float[][] matriz1, float[][] matriz2) {
		float[][] matrizRes = new float[3][3];

		for (int x = 0; x < 3; x++) {
			for (int y = 0; y < 3; y++) {
				matrizRes[x][y] = 0;
				for (int z = 0; z < 3; z++) {
					matrizRes[x][y] += matriz1[x][z] * matriz2[z][y];
				}
			}
		}
		printaMatriz(matrizRes);
	}
}
